`make proto` broken on master

**What happened?**: Due to a change in upstream dependencies, it looks like `make proto` is currently broken on master.

**How to reproduce it (as minimally and precisely as possible)?**: Run `make proto`, which will have this output at the end:

```
src/github.com/gengo/grpc-gateway/protoc-gen-grpc-gateway/main.go:20:2: use of internal package github.com/grpc-ecosystem/grpc-gateway/internal/codegenerator not allowed
src/github.com/gengo/grpc-gateway/protoc-gen-grpc-gateway/main.go:22:2: use of internal package github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway/internal/gengateway not allowed
The command '/bin/sh -c go get -u github.com/gengo/grpc-gateway/protoc-gen-grpc-gateway' returned a non-zero code: 1
make: *** [docker-build-proto] Error 1
```

**Anything else we need to know?**: Observed on both mac/docker for mac and ubuntu/minikube.

--- a/etc/proto/Dockerfile
+++ b/etc/proto/Dockerfile
@@ -12,7 +12,6 @@ RUN cp -r /include /bin
 
 RUN go get -u -v github.com/golang/protobuf/proto
 RUN go get -u -v github.com/gogo/protobuf/protoc-gen-gofast
-RUN go get -u github.com/gengo/grpc-gateway/protoc-gen-grpc-gateway
 RUN mkdir -p ${GOPATH}/src/github.com/pachyderm/pachyderm
 RUN date +%s >/last_run_time
 

--- a/etc/proto/run.sh
+++ b/etc/proto/run.sh
@@ -24,7 +24,6 @@ for i in src/**/*.proto; do \
         echo -e "\e[1;31mError:\e[0m missing \"go_package\" declaration in ${i}" >/dev/stderr
     fi
     protoc \
-        "-I${GOPATH}/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis" \
         "-I${GOPATH}/src/github.com/gogo/protobuf" \
         -Isrc \
         --gogofast_out=plugins=grpc,\

--- a/etc/proto/run.sh
+++ b/etc/proto/run.sh
@@ -19,7 +19,9 @@ fi
 go get github.com/gogo/protobuf/protoc-gen-gogofast
 
 cd "${GOPATH}/src/github.com/pachyderm/pachyderm"
-for i in src/**/*.proto; do \
+
+# shellcheck disable=SC2044
+for i in $(find src -name "*.proto"); do \
     if ! grep -q 'go_package' "${i}"; then
         echo -e "\e[1;31mError:\e[0m missing \"go_package\" declaration in ${i}" >/dev/stderr
     fi

